src/account-info/AccountDetailsScreen.js

```diff
@@ -37,13 +43,20 @@ class AccountDetailsScreen extends PureComponent<Props> {
         _: user.full_name || ' ',
       },
     };
+    let deactivatedText = null;
+    if (user.is_active === false) {
+      deactivatedText = (
+        <Label style={styles.deactivatedText} text="(This user/bot has been deactivated)" />
+      );
+    }
 
     return (
       <Screen title={title}>
         <AccountDetails user={user} />
+        {deactivatedText}
         <ZulipButton
           style={styles.pmButton}
-          text="Send private message"
+          text="Open chat"
```

Comment:
The webapp prefers "private messages", eschewing "chat" in this context (and almost all others); we should follow suit. Your original suggestion is probably a better option.

(Also, a new string will need to be added to `messages_en.json` regardless.)